File: java/StudentRegistration.java
import java.util.Scanner;

public class StudentRegistration {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("=============================================");
        System.out.println("   STUDENT REGISTRATION STATUS CHECKER");
        System.out.println("=============================================");

        System.out.print("\nEnter student name: ");
        String name = scanner.nextLine().trim();

        System.out.print("Enter number of registered units: ");
        int units = scanner.nextInt();

        String status;
        if (units > 7) {
            status = "Overload - Approval Required";
        } else {
            status = "Registration Accepted";
        }

        System.out.println("\n=============================================");
        System.out.println("           REGISTRATION SUMMARY");
        System.out.println("=============================================");
        System.out.println("  Student Name : " + name);
        System.out.println("  Units        : " + units);
        System.out.println("  Status       : " + status);
        System.out.println("=============================================");

        scanner.close();
    }
}

File: javascript/studentRegistration.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

function ask(question) {
  return new Promise((resolve) => rl.question(question, resolve));
}

async function checkRegistration() {
  console.log("=============================================");
  console.log("   STUDENT REGISTRATION STATUS CHECKER");
  console.log("=============================================");

  const name = (await ask("\nEnter student name: ")).trim();
  const unitsInput = await ask("Enter number of registered units: ");
  const units = parseInt(unitsInput);

  const status =
    units > 7 ? "Overload - Approval Required" : "Registration Accepted";

  console.log("\n=============================================");
  console.log("           REGISTRATION SUMMARY");
  console.log("=============================================");
  console.log(`  Student Name : ${name}`);
  console.log(`  Units        : ${units}`);
  console.log(`  Status       : ${status}`);
  console.log("=============================================");

  rl.close();
}

checkRegistration();
